thirdparty/drmingw_build_mingw.cmd

```diff
@@ -21,13 +21,15 @@ set MINGW64_FN=x86_64-6.3.0-release-win32-seh-rt_v5-rev1.7z
 set MINGW32_URL=https://sourceforge.net/projects/mingw-w64/files/Toolchains%%20targetting%%20Win32/Personal%%20Builds/mingw-builds/6.3.0/threads-win32/dwarf/%MINGW32_FN%/download
 set MINGW64_URL=https://sourceforge.net/projects/mingw-w64/files/Toolchains%%20targetting%%20Win64/Personal%%20Builds/mingw-builds/6.3.0/threads-win32/seh/%MINGW64_FN%/download
 
+REM  I downloaded these manually.

```

Comment:
What value does this addition bring? It's not ever clear to readers who "I" is.

Reply:
The downloads still work on AppVeyor, so I posit that there is a technical issue or incompatibility on your system that prevents these from being downloaded successfully.

Reply:
Sry. Left in by accident. It was a note to myself.

Reply:
I've seen a number of posts complaining about the sourceforge downloads. IDK what I could be doing wrong. The script is using very simple download tools that behave identically across systems. I'd vote for moving away from SF, if possible, but I don't have the resources to do anything about this.
Either way, I should have removed this message.